tests/test_creator/test_creator_delete_magic_attrs.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+
+import pop2net as p2n
+
+magic_agent_attributes = ["", "_assigned", "_id", "_position", "_head", "_tail"]
```

Comment:
This code will be run everytime vscode discovers tests. pytest fixtures are usually to way to do this.

Reply:
I rewrote this using a fixture.